Add topic about Reorder from the Storefront (#1260)
magento/merchdocs: src/customers/account-dashboard-reorders.md

---
title: Reorders
---

> For the reorder to be available, make sure the [_Allow Reorder_]({% link sales/reorders-allow.md %}) configuration option is enabled.

Reviewer: ```suggestion
Reorders are available to customers when the [_Allow Reorder_]({% link sales/reorders-allow.md %}) configuration option is enabled.
```

Author: Done ✅

A customer can initiate the reorder functionality for a specific order from two pages:

- My Orders page
- Order View page

## Reorder from My Orders page

The _Reorder_ button is displayed in the list with Orders near the _View_ button.

![]({% link images/images/account-dashboard-reorder.png %}){: .zoom}
_Reorder button on the My Order page_

**Case 1.** All products from the order are available for reorder

> User will be redirected to the [shopping cart]({% link quick-tour/shopping-cart.md %}), and all products are added to the cart

Reviewer: ```suggestion
Customer is redirected to the [shopping cart]({% link quick-tour/shopping-cart.md %}), and all products are added to the cart
```

Author: Done ✅

**Case 2.** Some/all products from the order are not available for reorder

The Reorder button will not appear on the _My Orders_ and _View Order_ pages.

![]({% link images/images/account-dashboard-reorder-grid.png %}){: .zoom}
_My Order page_

## Reorder from Order view page

**Case 1.** All products from the order are available for reorder

> User will be redirected to the cart, and all products are added to the cart

Reviewer: ```suggestion
Customer is redirected to the cart, and all products are added to the cart
```

Author: Done ✅

**Case 2.** Some/all products from the order are not available for reorder

> The Reorder button will not appear on the My Orders and View Order pages.

Reviewer: ```suggestion
The _Reorder_ button does not appear on the My Orders and View Order pages.
```

Author: Done ✅

{:.bs-callout-tip}

> If the cart is not empty and the user clicks Reorder (from the My Orders or Order View page), the existing products will remain in the cart with the added reorder products.

Reviewer: ```suggestion
If the cart is not empty and the customer clicks **Reorder** (from the My Orders or Order View page), the existing products remain in the cart with the added reorder products.
```

Author: Done ✅
